README.Rmd

```diff
@@ -400,17 +400,21 @@ To use sparklyr with Databricks Connect first launch a Cluster on Databricks. Th
     * Port (default port number is `15001`)
 
 To configure `sparklyr` with Databricks Connect, set the following environment variables:
-```{bash}
+```{bash, eval=FALSE}
 export SPARK_VERSION=2.4.4
 export SPARK_HOME=`databricks-connect get-spark-home`
 ```
 
 Now simply create a spark connection as follows
-```{r eval=FALSE}
-sc <- spark_connect(method = "databricks")
+```{r, eval=FALSE}
+spark_home <- system("databricks-connect get-spark-home")
+sc <- spark_connect(method = "databricks",
+                    spark_home = spark_home)
 copy_to(sc, iris, overwrite = TRUE)
 ```
 
+<img src="tools/readme/databricks-connect.png" class="screenshot" width=750 />
```

Comment:
Is this screenshot included in this PR?

Reply:
Yes indeed -- It's also shown in  https://github.com/yl790/sparklyr/tree/render_databricks_example_output#connecting-through-databricks-connect (the preview of what will be rendered after this change)